spec/models/task_spec.rb

```diff
@@ -16,4 +18,121 @@
       expect(task.slack_user_id).to eq("U1")
     end
   end
+
+  describe "#instigator_id" do
+    it "may have instigator" do
+      task = Task.new(
+        description: "hh",
+        user: users(:alex),
+        instigator: users(:tate),
+        list: lists(:outofdate),
+      )
+      expect(task.save).to eq(true)
+    end
+
+    it "may not have instigator" do
+      task = Task.new(
+        description: "hh",
+        user: users(:alex),
+        instigator: nil,
+        list: lists(:outofdate),
+      )
+      expect(task.save).to eq(true)
+    end
+  end
+
+  describe "user not mandatory" do
+    it "doesn't need users value to be full" do
```

Comment:
Update this to match the pattern above.
```
describe "#user_id" do
  it "may not have a user" do
    …
```

Reply:
Find at:
77261f210e33d61849c42ffc82a2c996db4a5bed